mappings/net/minecraft/block/AnvilBlock.mapping

```diff
@@ -11,8 +11,15 @@ CLASS bfz net/minecraft/block/AnvilBlock
 	METHOD <init> (Lbgs$c;)V
 		ARG 1 settings
 		ARG 2 stateBuilder
+	METHOD a configureEntity (Laox;)V
```

Comment:
I wonder if something like `configureFallingBlock` would be better? `Entity` is a little vague, though I realise it's obvious once you see the method signature.

Reply:
`configureFallingBlock` could be confusing as the class itself is called FallingBlock.

Reply:
Maybe not `configureFallingBlock`, but `configureEntity` is definitely too vague.

Reply:
Maybe `configureFallingBlockEntity`?

Reply:
I've changed it to `configureFallingBlockEntity`.